src/core/utilities/random.ts

```diff
@@ -0,0 +1,21 @@
+const alpha = "abcdefghijklmnopqrstuvwxyz0123456789";
+
+/**
+ * getRandomString() returns a randomly generated alphanumeric string of the
+ * given length. If no length is provided, the result will be 32 characters.
+ * Note that this function does not use cryptographically secure randomness and
+ * as such should not be used for cryptographic purposes.
+ *
+ * @param length Optional. The length of the string to generate
+ * @returns      A randomly generated alphanumeric string
+ */
+export const getRandomString = (length?: number): string => {
```

Comment:
for fun: 
const randomHexString = length => [...Array(length)].map(() => _.random(16).toString(16)).join('');

but perhaps we should call this file getRandomString instead of random.

Reply:
My thought with the filename was that we may have more random utility functions to put here in the future.

Reply:
maybe call it utility?

Reply:
I removed it completely given that we're switching to content hashes as filenames